databuilder/databuilder/models/column_usage_model.py

```diff
@@ -1,118 +0,0 @@
-# Copyright Contributors to the Amundsen project.
-# SPDX-License-Identifier: Apache-2.0
-
-from typing import Iterator, Union
```

Comment:
make sense to delete the duplicate one... I kinda recalled now which the model is purely for csv column usage support (e.g https://github.com/amundsen-io/amundsendatabuilder/pull/173)